### PR TITLE
⬆️ upgrade to antd config provider

### PR DESCRIPTION
**Pull request by jstorm31**
Resolves issue #15

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export interface Action {
 }
 
 export interface LocaleData<T = {}> {
-    [locale: string]: object & T;
+    [locale: string]: T;
```
Comment (jstorm31) [inline src/types.ts]: Is this edit valid? It complained about incompatible types and I'm not confident in typescript at all.

Comment (jancama2) [reply src/types.ts]: I guess so, I had the same problem. @horaklukas could now since it is his work :D

Comment (horaklukas) [reply src/types.ts]: Looks valid, i tried and didn't get any errors or warnings from build neither from type check or lint..

Comment (jstorm31) [reply src/types.ts]: Yes, I can also confirm the build proceeds fine for me. Good 👌

**Pull request merged**